src/main/kotlin/no/nav/familie/tilbake/config/OppdragMQConfig.kt

```diff
@@ -0,0 +1,65 @@
+package no.nav.familie.tilbake.config
+
+import com.ibm.mq.constants.CMQC
+import com.ibm.mq.jms.MQQueueConnectionFactory
+import com.ibm.msg.client.jms.JmsConstants
+import com.ibm.msg.client.wmq.common.CommonConstants
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.autoconfigure.jms.DefaultJmsListenerContainerFactoryConfigurer
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.jms.config.DefaultJmsListenerContainerFactory
+import org.springframework.jms.config.JmsListenerContainerFactory
+import org.springframework.jms.connection.JmsTransactionManager
+import org.springframework.jms.connection.UserCredentialsConnectionFactoryAdapter
+import javax.jms.ConnectionFactory
+import javax.jms.JMSException
+
+private const val UTF_8_WITH_PUA = 1208
+
+@Configuration
+class OppdragMQConfig(@Value("\${oppdrag.mq.hostname}") val hostname: String,
+                      @Value("\${oppdrag.mq.queuemanager}") val queuemanager: String,
+                      @Value("\${oppdrag.mq.channel}") val channel: String,
+                      @Value("\${oppdrag.mq.port}") val port: Int,
+                      @Value("\${oppdrag.mq.user}") val user: String,
+                      @Value("\${oppdrag.mq.password}") val password: String) {
```

Comment:
Brude kanskje vært ConfigurationProperties:

@ConfigurationProperties("oppdrag.mq")
@ConstructorBinding
class OppdragMQConfig(val hostname: String,
                      val queuemanager: String,
                      val channel: String,
                      val port: Int,
                      val user: String,
                      val password: String) {

Reply:
skulle gjerne gjort det men da klager intellij/spring på `No qualifying bean of type 'java.lang.String'`